src/main/java/wolox/training/exceptions/BookIdMismatchException.java

```diff
@@ -0,0 +1,9 @@
+package wolox.training.exceptions;
+
+public class BookIdMismatchException extends RuntimeException{
```

Comment:
The best thing would be to indicate what return status this exception should have when it appears.
To do this we add the annotation @ResponseStatus. In this case, I would put a status of 400 since the ids entered do not match.

To have a little more information on the subject of states I leave you the following [link](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status).

Reply:
I added @ResponseStatus in this part.